src/user/profile/content/PostContent.js

```diff
@@ -0,0 +1,149 @@
+import React, { Component } from "react";
+import { Button, Modal, Form, Col } from "react-bootstrap";
+import "./PostContent.scss";
+import { connect } from "react-redux";
+import parse from "html-react-parser";
+import { getAllCommentsOfPost } from "../../../actions/commentAction";
+import userIcon2 from '../../../images/userIcon2.jpg'
+
+class PostContent extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      post: {},
+      commentsLoaded: false,
+      postId: "",
+    };
+  }
+
+  componentDidMount() {
+    setTimeout(() => {
+      this.props.getAllCommentsOfPost(this.props.postId);
+    }, 2000);
+  }
+
+  componentWillReceiveProps(nextProps) {
+    console.log(nextProps);
+    const { displayingPost, posts, comments } = nextProps;
+    const post = posts[displayingPost];
+
+    this.setState(
+      {
+        post: post,
+        comments: comments,
```

Comment:
Please add `comments: null` in the state in the constructor. Please try not to add new properties to state without declaring them in the constructor.